<launch>
	<!-- Launch an ROI0 node and pass any necessary parameters -->
	<arg name="sub_topic" default="/camera/color/image_raw" />
	<arg name="pub_topic" default="/camera/color/image_interest" />
	<node name="apply_filter" pkg="lab2" type="ROI.py" output="screen">
		<param name="sub_topic" value="$( arg sub_topic )" />
		<param name="pub_topic" value="$( arg pub_topic )" />
	</node>
</launch>